index.bs

```diff
@@ -12032,22 +12034,18 @@ defined in section [[#platformobjectset]].
 </div>
 
 
-<h4 id="getownproperty">Platform object \[[GetOwnProperty]] method</h4>
+<h4 id="legacy-platform-object-getownproperty" oldis="getownproperty">Legacy platform object \[[GetOwnProperty]] method</h4>
```

Comment:
Fixed the typo.

> Guess it's up to you to decide whether it's worth it.

You have a lot more experience than I, here. What do you think is the best solution?